appveyor.yml

```diff
@@ -2,6 +2,7 @@ version: '{build}'
 branches:
   only:
   - master
+  - appveyor
```

Comment:
it should build pr's if the webhook is configured right, there may have just been a delivery failure?

I think without a regex this would need to be an exact match anyway